cmd/config/internal/commands/run-fns.go

```diff
@@ -65,6 +66,8 @@ func GetRunFnRunner(name string) *RunFnRunner {
 	r.Command.Flags().StringArrayVar(
 		&r.Mounts, "mount", []string{},
 		"a list of storage options read from the filesystem")
+	c.Flags().StringSliceVar(&r.InputPattern, "input-pattern", kio.DefaultMatch,
```

Comment:
Should we have this as a flag? I guess the behavior should be static whether to accept certain file formats or not. Other tools like terraform and cloudformation support both formats but user need not specify them through flags. Adding more such flags might make them less discoverable IMO. WDYT?

Reply:
I'd like to know more about why folks need to use json files before supporting them by default.  The flag gives folks the ability to opt-in, without it being on by default.  The JSON experience is not going to be as good as the YAML experience -- folks should probably just convert their JSON files to YAML files.